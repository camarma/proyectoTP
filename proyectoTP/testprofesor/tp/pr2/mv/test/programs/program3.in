apila 65
push 65
out
halt
end
