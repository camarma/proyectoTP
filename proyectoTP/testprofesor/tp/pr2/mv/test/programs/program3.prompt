ejecuta 3
step 3
quit