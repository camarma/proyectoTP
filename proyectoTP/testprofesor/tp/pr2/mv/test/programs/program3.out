Introduce el programa fuente
> Error: Instruccion incorrecta
> > > > El programa introducido es:
0: PUSH 65
1: OUT
2: HALT
> Error: Comando no reconocido
> Comienza la ejecucion de  PUSH 65
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 65
Comienza la ejecucion de  OUT
A
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: <vacia>
Comienza la ejecucion de  HALT
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: <vacia>
> 